tracer/build/_build/PrepareRelease/GenerateIntegrationDefinitions.cs

```diff
@@ -10,12 +10,17 @@
 using System.Reflection;
 using System.Runtime.InteropServices;
 using System.Runtime.Loader;
+using Microsoft.VisualStudio.Services.WebApi;
+using Newtonsoft;
+using Newtonsoft.Json;
```

Comment:
You _serialized_ with System.Text.Json, but then _deserialized_ with Newtonsoft.Json... You're probably better off using the same for both
```suggestion
```

Reply:
Yep, deserialization with `System.Text.Json` was throwing exceptions (because of the version tuples) and changed to Newtonsoft because of that. It's true I could move to Newtonsoft for the serialization also

Reply:
Yes, please, let's not mix serializers.

Reply:
Yeah, going to reinforce, definitely don't mix serializers, sorry 😄

Reply:
Moved everything to Newtonsoft